pkg/odo/genericclioptions/context.go

```diff
@@ -28,12 +28,16 @@ const (
 )
 
 // NewContext creates a new Context struct populated with the current state based on flags specified for the provided command
-func NewContext(command *cobra.Command, ignoreMissingConfiguration ...bool) *Context {
+func NewContext(command *cobra.Command, toggles ...bool) *Context {
```

Comment:
That's a bad API… this idiom only works for an optional bool. In this case, the calling code has no idea what is expected without reading the code of this function, especially considering that the different values that can be passed are not documented.

Reply:
yes, we can document this or have an options list. This was done in a hurry